src/util.js

```diff
@@ -503,7 +503,7 @@ export default {
 
     if (_self.msCrypto) {
       return _self.msCrypto.subtle;
-    } else if (_self.crypto) {
+    } else if (_self.crypto && _self.crypto.webkitSubtle) {
```

Comment:
@tanx i've added this && in order to be sure that _self.crypto.webkitSubtle exists before returning it;

this is required supposing that a browser different fomr safary could not offer the standard API but have the _self.crypto object; by the current code that browser was returning blindly webkitSubtle.